src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBTriggerListener.cs

```diff
@@ -10,31 +10,39 @@
 using Microsoft.Azure.Documents.Client;
 using Microsoft.Azure.WebJobs.Host.Executors;
 using Microsoft.Azure.WebJobs.Host.Listeners;
+using Microsoft.Extensions.Logging;
 
 namespace Microsoft.Azure.WebJobs.Extensions.CosmosDB
 {
     internal class CosmosDBTriggerListener : IListener, IChangeFeedObserverFactory
     {
         private readonly ITriggeredFunctionExecutor executor;
-        private readonly ChangeFeedEventHost host;
+        private readonly ILogger logger;
```

Comment:
nit: might consider making the field naming consistent with other files (_ prefix)

Reply:
Done